examples/nextjs-with-supabase-auth/pages/_app.js

```diff
@@ -1,13 +1,12 @@
-import { UserProvider } from '@supabase/supabase-auth-helpers/react'
-import { supabaseClient } from '@supabase/supabase-auth-helpers/nextjs'
-import './../style.css'
+// pages/_app.js
+import React from 'react';
+import { UserProvider } from '@supabase/auth-helpers-react';
+import { supabaseClient } from '@supabase/auth-helpers-nextjs';
 
-export default function MyApp({ Component, pageProps }) {
+export default function App({ Component, pageProps }) {
   return (
-    <main className={'dark'}>
```

Comment:
Any particular reason why this div with the dark class was removed?

Reply:
Yes, the dark background was affecting the readability after signup.